internal/observability/client.go

```diff
@@ -40,7 +40,9 @@ func Init(ctx context.Context, serviceName, serviceVersion string, config Config
 		return nil
 	}
 
-	logger.Debugf("OTEL is enabled, exporting to %s", os.Getenv("OTEL_EXPORTER_OTLP_ENDPOINT"))
+	logger.Debugf("OTEL is enabled, exporting to %s at log level %s",
+		os.Getenv("OTEL_EXPORTER_OTLP_ENDPOINT"),
```

Comment:
do you want this extra newline here?

Reply:
I generally find things like this improve readability, but we also don't do that anywhere else in the codebase, so I'll just remove it. Probably not very Golang of me :p